SampleApp/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,5 +1,15 @@
 {
   "images" : [
+    {
```

Comment:
I guess that's just what Xcode generates these days 😆

Reply:
Seems that way, sorry!

Reply:
Don't worry, can stay I think!